### PR TITLE
Implement java.io.ObjectStreamException.

### PR DESCRIPTION
**Pull request by andreaTP**
this will be easy

### CODE REVIEW

File: javalib/src/main/scala/java/io/Throwables.scala [hunk 1]
```diff
@@ -17,3 +17,7 @@ class UTFDataFormatException(s: String) extends IOException(s) {
 class UnsupportedEncodingException(s: String) extends IOException(s) {
   def this() = this(null)
 }
+
+abstract class ObjectStreamException(s: String) extends IOException(s) {
+  def this() = this(null)
```
Comment (sjrd) [inline javalib/src/main/scala/java/io/Throwables.scala]: The two constructors should be `protected`, according to [the JavaDoc](http://docs.oracle.com/javase/7/docs/api/java/io/ObjectStreamException.html)

**Comment by sjrd** [comment]
That's all.

Although I fail to see how adding an `abstract class` with no implementation can be any good?

**Comment by andreaTP** [comment]
fixed, just a placeholder for cross compilation

**Comment by sjrd** [comment]
LGTM

Jenkins, test this please

**Comment by sjrd** [comment]
test this please

**Comment by nicolasstucki** [comment]
Jenkins, test this please

**Comment by nicolasstucki** [comment]
oops, we asked with 2 sec of difference ;)

**Comment by sjrd** [comment]
test this please

**Pull request merged**